swift-ci/master/ubuntu/22.04/Dockerfile

```diff
@@ -75,9 +77,18 @@ RUN set -e; \
     && tar -xzf swift.tar.gz --directory $SWIFT_PREFIX --strip-components=1 \
     && chmod -R o+r $SWIFT_PREFIX/usr/lib/swift \
     && rm -rf "$GNUPGHOME" swift.tar.gz.sig swift.tar.gz \
+    # - Install wamr WebAssembly runtime under /usr/local/ for testing
+    && WAMR_BUILD_DIR=/tmp/wamr-build \
+    && WAMR_SOURCE_TARBALL_URL="https://github.com/bytecodealliance/wasm-micro-runtime/archive/refs/tags/WAMR-${SWIFT_CI_WAMR_VERSION}.tar.gz" \
+    && mkdir -p "$WAMR_BUILD_DIR" \
+    && curl -L "$WAMR_SOURCE_TARBALL_URL" | tar -xz --strip-component 1 -C "$WAMR_BUILD_DIR" \
+    && cmake -G Ninja -B "$WAMR_BUILD_DIR" -DWAMR_BUILD_AOT:BOOL=NO "$WAMR_BUILD_DIR/product-mini/platforms/linux/" \
+    && cmake --build "$WAMR_BUILD_DIR" \
+    && cmake --install "$WAMR_BUILD_DIR" --prefix "$SWIFT_CI_WAMR_PREFIX" \
+    && rm -rf "$WAMR_BUILD_DIR" \
```

Comment:
Is there a reason this is happening in Docker image compared to build-script?

Reply:
Building it in build-script makes more sense 👍